src/components/FilterGroup.tsx

```diff
@@ -119,7 +120,12 @@ function CheckboxOptions({
 }) {
   const searchUtilities = useSearchUtilities();
   const { searchValue } = useFilterGroupContext();
-  const shouldRenderOption = useCallback((option: { displayName: string, value: unknown }) => {
+  const shouldRenderOption = (option:
```

Comment:
@oshi97 do we not want to wrap this in a `useMemo` or a `useCallback`? I'm wondering if, as is, we could erroneously re-render if `searchValue` and `searchUtilities` remained the same.